2-ui/4-forms-controls/1-form-elements/article.md

```diff
@@ -8,7 +8,7 @@ Working with forms will be much more convenient when we learn them.
 
 Document forms are members of the special collection `document.forms`.
 
-That's a so-called "named collection": it's both named and ordered. We can use both the name or the number in the document to get the form.
+That's a so-called **"named collection"**: it's both named and ordered. We can use both the name or the number in the document to get the form.
```

Comment:
You make it bold this way. Are you sure? Maybe should be italic (single `*`)?

Reply:
Done.